Documentation/teaching/info/vm.rst

```diff
@@ -4,118 +4,118 @@
+   qemu-system-i386: -chardev pty,id=virtiocon0: char device redirected to /dev/pts/68 (label virtiocon0)
+
+.. note:: To show the QEMU console use ``QEMU_DISPLAY=sdl make
+          boot```. This will show the VGA output and will also give
           access to the standard keyboard.
 
+.. note:: The virtual machine setup scripts and configuration files are located
+          in ``tools/labs/qemu/``.
+
+.. note:: The kernel image is only built the first time the virtual machine is
+          started. It will be rebuilt each time there are updates to local
+          kernel source code.
+
 .. _vm_interaction_link:
 
-Connecting to the VM
---------------------
+Connecting to the Virtual Machine
+---------------------------------
 
-Once the machine is booted you can connect to it on the serial port. A
-link named *serial.pts* is created to the right emulated serial port
-and you can use **minicom**, **picocom** to connect to the virtual
-machine from the host:
+Once the virtual machine is started you can connect to it on the serial port. A
+symbolic link named ``serial.pts`` is created to the emulated serial port
+device:
 
 .. code-block:: shell
 
-   $ minicom -D serial.pts
+   .../linux/tools/labs$ ls -l serial.pts
+   lrwxrwxrwx 1 razvan razvan 11 Apr  1 08:03 serial.pts -> /dev/pts/68
 
-   Poky (Yocto Project Reference Distro) 2.3 qemux86 /dev/hvc0
-
-   qemux86 login: root
-   root@qemux86:~#
-
-Networking is also setup and you can use ssh to connect to the virtual
-machine after finding out the allocated IP address:
+On the host you use the ``minicom`` command to connect to the virtual machine
+via the ``serial.pts`` link:
 
 .. code-block:: shell
 
-   $ mincom -D serial.pts
-
+   .../linux/tools/labs$ minicom -D serial.pts
+   [...]
    Poky (Yocto Project Reference Distro) 2.3 qemux86 /dev/hvc0
 
    qemux86 login: root
-   root@qemux86:~# ifconfig
-   eth0      Link encap:Ethernet  HWaddr 52:54:00:12:34:56
-             inet addr:172.20.0.6  Bcast:172.20.0.255  Mask:255.255.255.0
-	     UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
-	     RX packets:41 errors:0 dropped:0 overruns:0 frame:0
-	     TX packets:6 errors:0 dropped:0 overruns:0 carrier:0
-	     collisions:0 txqueuelen:1000
-	     RX bytes:7578 (7.4 KiB)  TX bytes:1296 (1.2 KiB)
-
-   lo        Link encap:Local Loopback
-	     inet addr:127.0.0.1  Mask:255.0.0.0
-	     inet6 addr: ::1%134535719/128 Scope:Host
-	     UP LOOPBACK RUNNING  MTU:65536  Metric:1
-	     RX packets:0 errors:0 dropped:0 overruns:0 frame:0
-	     TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
-	     collisions:0 txqueuelen:1000
-	     RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
-
-   $ ssh root@172.20.0.6
```

Comment:
Maybe move the ssh part in another section, I think it is worth keeping it.

Reply:
It complicates the document and requires additional actions, another image. It should be part of a different page: "Extending the Virtual Machine Deployment".

Reply:
Fair, that is a good alternative.